src/lib/y2storage/disk_analyzer.rb

```diff
@@ -70,98 +72,28 @@ class DiskAnalyzer
 
     DEFAULT_DISK_CHECK_LIMIT = 10
 
-    # @return [Array<String>] device names of installation media.
-    #       Filled by #analyze.
-    attr_reader :installation_disks
-
-    # @return [Array<String>] device name of disks to install on.
-    #       Filled by #analyze.
-    attr_reader :candidate_disks
-
-    # @return [Hash{String => Array<::Storage::Partition>}] Linux
-    #     partitions found in each candidate disk. Filled by #analyze.
-    #     @see #find_linux_partitions
-    attr_reader :linux_partitions
-
-    # @return [Hash{String => Array<::Storage::Partition>}] MS Windows
-    #     partitions found in each candidate disk.
-    #     Filled by #analyze only if #linux_partitions is empty.
-    #     @see #find_windows_partitions
-    attr_reader :windows_partitions
-
-    # @return [Hash{String => Array<::Storage::Partition>}] EFI partitions
-    #     found in each candidate disk. Filled by #analyze.
-    #     @see #find_efi_partitions
-    attr_reader :efi_partitions
-
-    # @return [Hash{String => Array<::Storage::Partition>}] PReP partitions
-    #     found in each candidate disk. Filled by #analyze.
-    #     @see #find_prep_partitions
-    attr_reader :prep_partitions
-
-    # @return [Hash{String => Array<::Storage::Partition>}] GRUB partitions
-    #     found in each candidate disk. Filled by #analyze.
-    #     @see #find_grub_partitions
-    attr_reader :grub_partitions
-
-    # @return [Hash{String => Array<::Storage::Partition>}] Swap partitions
-    #     found in each candidate disk. Filled by #analyze.
-    #     @see #find_swap_partitions
-    attr_reader :swap_partitions
-
-    # @return [Hash{String => Y2Storage::DiskSize}] MBR gap sizes
-    #     found on each candidate disk. Filled by #analyze.
-    #     @see #find_mbr_gap
-    attr_reader :mbr_gap
-
-    # @return [Fixnum] Maximum number of disks to check.
-    #     @see #find_installation_disks
-    attr_accessor :disk_check_limit
-
-    def initialize
-      Yast.import "Arch"
-
-      @installation_disks = []
-      @candidate_disks    = []
-      @linux_partitions   = {}
-      @windows_partitions = {}
-      @efi_partitions     = {}
-      @prep_partitions    = {}
-      @grub_partitions    = {}
-      @swap_partitions    = {}
-      @mbr_gap            = {}
-
-      # Maximum number of disks to check. This might be important on
-      # architectures that tend to have very many disks (s390).
-      @disk_check_limit = DEFAULT_DISK_CHECK_LIMIT
-    end
-
-    # Analyze disks and partitions. Make sure to call this before querying
-    # any member variables.
+    # @return [Fixnum] Maximum number of disks to check with "expensive"
+    # operations.
```

Comment:
Indent "operations" by 2 spaces, otherwise yardoc misparses it and considers it independent from the "@return"